example_test.go

```diff
@@ -237,3 +238,57 @@ func ExampleCredential() {
 		panic(err)
 	}
 }
+
+func ExampleTrace() {
+	// trace.ExecutableTrace is used to define trace hooks that track the
+	// execution of local credential helper executables, and it works with
+	// the credentials.Store interface. The defined hooks will run whenever
+	// a credential helper executable is executed. Multiple ExecutableTrace
+	// can be defined and used, and recently added ExecutableTrace is called
+	// first.
+	store, err := credentials.NewStore("example/path/config.json", credentials.StoreOptions{})
+	if err != nil {
+		panic(err)
+	}
+
+	// define two ExecutableTrace, and trace2 will be called before trace1.
+	// The 'action' argument refers to one of 'store', 'get' and 'erase' defined
+	// by the docker credential helper protocol.
+	//
+	// Reference:
+	//   - https://docs.docker.com/engine/reference/commandline/login/#credential-helper-protocol
```

Comment:
I don't think it's necessary to demonstrate calling `WithExecutableTrace` multiple times.